pytripgui/settings.py

```diff
@@ -84,6 +102,15 @@ def get_value(self, query):
                 return None
         return temp
 
+    def get_value_str(self, query):
+        """ Wrapper which will return empty string if key is not found.
+        """
+        # TODO: better to test if temp is string, and if not return ""
+        temp = self.get_value(query)
```

Comment:
I think I have to do a wrapper no matter what, because I am not sure if anything relies on a None return from the ```self.get_value()``` method. Note also that self.get_value() is doing a bit more than just a dict lookup.